src/pages/Checklist/index.tsx

```diff
@@ -236,14 +241,34 @@ export const Checklist = () => {
               <Text className="checkbox-label" style={{ display: 'inherit' }}>
                 For Ubuntu 20.04:
                 <pre className="my0">
+                  {'\n'}
+                  <span style={{ color: colors.red.medium }}>
+                    sudo apt install chrony
```

Comment:
@PatrickAlphaC  I have second thought: the context is about Ubuntu 20.04, so with the default `timesyncd` service it should be fine to sync the time.

Reply:
How can we check that is working? I used a new Ubuntu install and was not able to `set-ntp`

Reply:
Is your Ubuntu version 20.04?

You can run this command to see the status of the Ubuntu default `systemd-timesyncd` NTP client:
```sh
sudo systemctl status systemd-timesyncd.service
```

If it's inactive, you can (1) try to restart it:
```sh
sudo systemctl restart systemd-timesyncd.service
```

or (2) install the alternative `chrony` or `ntp` pacakges.

Reply:
`Failed to restart systemd-timesyncd.service: Unit systemd-timesyncd.service is masked.`

So I did (2) hence why I made this issue / PR